source/tutorials/utilities.rst

```diff
@@ -177,6 +177,78 @@ run, using ``inspect-metadata``, we can learn all about it:
 
    How many IDs are there? How many columns? Are there any categorical columns? Why?
 
+Casting Metadata
+................
+
+In the :doc:`Metadata tutorial <metadata>` we learned about column types and utilizing the 
+``qiime tools cast-metadata`` tool to specifiy column types within a provided metadata file.
+Below we will go through a few scenarios of how this tool can be used, and some common mistakes
+that may come up.
+
+We'll start by first downloading some sample metadata. **Note**: This is the same sample
+metadata used in the **Inspect Metadata** section, so you can skip this step if you have
+already downloaded the ``sample_metadata.tsv`` file from above.
+
+.. download::
+   :url: https://data.qiime2.org/2021.8/tutorials/pd-mice/sample_metadata.tsv
+   :saveas: sample_metadata.tsv
+
+In this example, we will cast the ``days_post_transplant`` column from ``numeric`` to ``categorical``,
+and the ``mouse_id`` column from ``categorical`` to ``numeric``. The rest of the columns contained
+within our metadata will be left as-is.
+
+.. command-block::
+   :stdout:
+
+   qiime tools cast-metadata sample_metadata.tsv \
+   --cast days_post_transplant:categorical --cast mouse_id:numeric
+
+If the ``--output-file`` flag is enabled, the specified output file will contain the modified column types that we cast above
+for the original ``sample_metadata.tsv``.
+
+If you do not wish to save your casted metadata to an output file, you can omit the ``--output-file`` parameter and
```

Comment:
`s/casted/cast/g`

Reply:
@thermokarst can you clarify what your requested change is for this code block? I'm not sure I understand what `s/casted/cast/g` is meant to replace.

Reply:
Sure - my bad, we chatted about that syntax a while ago, I assumed you remembered it - it means "substitute the first value with the second value, everywhere" - its really common syntax, you'll see it in lots of places like sed, vim, your favorite editor, etc.

Reply:
Sorry, I should clarify - its commonly-adopted syntax that you'll see pop up in lots of places, so it'll be good to file this one away because I bet you'll see it many more times!

Reply:
Ahhh, okay gotcha! I do remember seeing this in vim, but I didn't realize this was common shorthand outside of that. Thanks!